synapse/lib/layer.py

```diff
@@ -1772,6 +1772,7 @@ async def pack(self):
     async def truncate(self):
         '''
         Nuke all the contents in the layer, leaving an empty layer
+        NOTE: This internal API is deprecated but is kept for Nexus event backward compatibility
```

Comment:
Can we add a s_common.deprecated call here, so at least someone can know if its been used?